client/telemetry/src/lib.rs

```diff
@@ -121,25 +121,7 @@ impl TelemetryWorker {
 	/// Instantiate a new [`TelemetryWorker`] which can run in background.
 	///
 	/// Only one is needed per process.
-	pub fn new(buffer_size: usize) -> Result<Self> {
```

Comment:
Why did you removed this?

This was added by intention, because 99% of the people don't need to provide any transport.

Reply:
I didn't remove it, just added the `Option`. Seemed dumb to have two methods that are 99% identical.

Reply:
That is right, but than you make both use a common function.